internal/nodes/avi_model_evh_nodes.go

```diff
@@ -910,6 +910,12 @@ func (o *AviObjectGraph) BuildPolicyPGPoolsForEVH(vsNode []*AviEvhVsNode, childN
 			if servers := PopulateServersForNPL(poolNode, namespace, path.ServiceName, true, key); servers != nil {
 				poolNode.Servers = servers
 			}
+		} else if serviceType == lib.NodePort &&
```

Comment:
Do we need a check if service type is not node port in MCI mode ? 
otherwise we would select the else condition in line no: 923

Reply:
I have made the changes to handle non NodePort mode.